.github/workflows/ghcr.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   push-to-ghcr:
     name: Build and push image for commit
     runs-on: ubuntu-latest
-    if: github.repository == 'WordPress/openverse-frontend'
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.owner.login == 'WordPress'
```

Comment:
You can see an example of the `github.event.pull_request` object in the [GitHub PR docs](https://docs.github.com/en/rest/pulls/pulls#get-a-pull-request).